src/main/scala/mesosphere/marathon/core/group/impl/GroupManagerImpl.scala

```diff
@@ -43,9 +44,25 @@ class GroupManagerImpl(
     * Even though updates go through the workqueue, we want to make sure multiple readers always read
     * the latest version of the root. This could be solved by a @volatile too, but this is more explicit.
     */
-  private[this] val root = LockedVar(initialRoot)
-
-  override def rootGroup(): RootGroup = root.get()
+  private[this] var root = LockedVar(initialRoot)
```

Comment:
Mutable reference to the mutable state is a little bit too much :)
What do think about using `val` with `LockedVar` or to remove `LockedVar` and use `@volatile`?

Reply:
Yeah, it slipped through the cracks, while I was experimenting (with `@volatile` included). It was `val` before. Thanks for catching it.